types/scrivito/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for Scrivito 1.16
```

Comment:
Can you lowercase the package name?

@sandersn should the header parser be stricter about this?

Reply:
I'll do it anyway for correctness